docs/content/docs/api-reference/workflow-api/world.mdx

```diff
@@ -0,0 +1,149 @@
+---
+title: World helpers
+---
+
+# Accessing the World singleton
+
+> “Is there any public API to access the active default World? I'd like to be able to list active runs in my app, like I can via the CLI. I could store them myself, but if there is an API for this that works across testing and deployment, that'd be even better!”
+
+Yes. Import `getWorld` or any of the helper functions below directly from `workflow/api`. They proxy to the same singleton that powers the CLI, whether you're running locally with the embedded world, in tests (via `setWorld`), or on Vercel.
+
+```typescript
+import {
+  getWorld,
+  listRuns,
+  listSteps,
+  cancelRun,
+  queue,
+} from 'workflow/api';
+
+const runs = await listRuns({ status: 'running', pagination: { limit: 10 } });
+await cancelRun(runs.data[0].runId);
+
+// You still have full access to the underlying World instance when needed.
+const world = getWorld();
+await world.start?.();
+```
+
+## Example: List and cancel runs in a Next.js route
+
+```typescript filename="app/api/workflows/route.ts"
+import { listRuns, cancelRun } from 'workflow/api';
+
+export async function GET() {
+  const runs = await listRuns({ status: 'running', pagination: { limit: 20 } });
+  return Response.json({ runs: runs.data });
+}
+
+export async function POST(request: Request) {
+  const { runId } = await request.json();
+  await cancelRun(runId);
+  return Response.json({ ok: true });
+}
+```
+
+This works the same way in development and production—just make sure the usual `WORKFLOW_*` environment variables are present so the correct world implementation can be chosen.
+
+## Helper catalog
```

Comment:
I think we might want to generate these from the interface, so they don't get outdated. For now, we could just link to the latest interface.ts file in the npm package?